src/components/Form.tsx

```diff
@@ -43,7 +42,7 @@ export class UnwrappedForm extends Component<IFormWrappedProps> {
         defaults,
         model,
         onSave,
-        onSuccess: onCancel,
+        onSuccess: this.onSuccess,
```

Comment:
I guess it's fine that `this.onSuccess()` returns undefined if both are not set ?

Reply:
Yeah, onCancel and onSuccess are both void functions. this.onSuccess doesn't return anything no matter what.